Source/WebKit/UIProcess/API/Cocoa/WKWebExtension.mm

```diff
@@ -510,4 +510,16 @@ - (BOOL)_hasModularBackgroundContent
 
 #endif // ENABLE(WK_WEB_EXTENSIONS)
```

Comment:
should be `!ENABLE(WK_WEB_EXTENSIONS)`

Reply:
Sorry, not sure I'm understanding -- isn't the initial condition `#if ENABLE(WK_WEB_EXTENSIONS`? Why would we want to invert the condition on the closing `#endif`?

Reply:
Usually we do something like `#endif // !ENABLE(WK_WEB_EXTENSIONS)` if we have an `#if`, `#else` and then `#endif` to indicate that the second block is for the `not` case. but actually, i think because there's the `// ENABLE(WK_WEB_EXTENSIONS)` with the `#else` block, `ENABLE(WK_WEB_EXTENSIONS)` is actually correct. so you can disregard this comment.